app/scripts/components/facets/templates/facet.html

```diff
@@ -70,6 +71,16 @@
           {{ ::term.key | translate }}
         </span>
       </span>
+      <span data-ng-if="term.key === '_missing'"
```

Comment:
If we did this on line 71 and removed the markup tailored to 'MISSING', we'd end up with a lot of ternary expressions wherever `term.key` is referenced. For example, `{{ (term.key === '_missing' ? 'No Data' : term.key) | translate }}` or `data-ng-click="tc.add(tc.name, term.key === '_missing' ? 'MISSING' : term.key)"` Seems to be adding a lot of more noise than having a dedicated markup for `_missing`. Thoughts?